### ISSUE TITLE
Convert : to - in template recommendations

### ISSUE THREAD

**Issue by wesruv** [backdrop/backdrop-issues#1706]
Having a colon in the file name acts pretty funk in command line and my editor of choice hates it.

The `Powered by` block is coming through with a colon in it's template recommendation, think we should convert it to a dash:
```
<!-- FILE NAME SUGGESTIONS:
   * block--system--powered-by.tpl.php
   * block--system.tpl.php
   x block--system:powered-by.tpl.php
   * block.tpl.php
-->
```

**Comment by gifad** [comment]
Yes, but do we really need this suggestion (based on `$block->plugin`) when we already have a suggestion based on `$block->module`--`$block->delta` ?

**Comment by wesruv** [comment]
Ha, I didn't even notice the first recommendation in that list :P

Regardless, `:` should probably be disallowed in template recommendations whether or not we need `$block->plugin` recommendations.
